src/odemis/acq/move.py

```diff
@@ -570,19 +570,23 @@ def _update_conversion(self):
         # the inverse transformation is used for sample stage -> stage
 
         # pre-tilt is rotation around the stage-bare x axis
-        r = self.pre_tilt
-        # note: this is currently for tfs, which does not have scale, shear or translation
+        rx = self.pre_tilt
 
+        # note: this is currently for tfs, which does not have scale, shear or translation
         # rotation around x axis: fm = tf, sem = tf_inv
-        tf, tf_inv = _get_transforms(r)
+        tf, tf_inv = get_rotation_transforms(rx=rx)
 
-        # TODO: enable scan rotation once complete
-        sr_matrix, sr_matrix_inv = self._get_scan_rotation_matrix()
+        # NOTE: not yet implemented, see meteor-1100-fibsem-tab
+        sr = 0
+        if not self.use_scan_rotation:
+            sr = 0
 
-        logging.debug(f"tf_matrix: {tf}, sr_matrix: {sr_matrix}")
+        # get scan rotation matrix (rz -> rx)
+        tf_sr, tf_inv_sr = get_rotation_transforms(rx=rx, rz=sr)
+        logging.debug(f"tf_matrix: {tf}, tf_sr: {tf_sr}")
 
-        tf_sr = tf.dot(sr_matrix)
-        tf_inv_sr = sr_matrix_inv.dot(tf_inv)
+        # tmp: assert that scan rotation matrix has no impact on standard tf, remove once scna rotation implemented
+        numpy.testing.assert_array_equal(tf, tf_sr)
```

Comment:
I don't think you need to check for this every time it's computed. The code uses explicitly `rz=0`. Looking at the code of `get_rotation_transforms()`, you can see it's converted into an identity matrix. So the 2 matrices will be identical, for sure. Let's drop this assert code.